cli/src/lib/libDefs.js

```diff
@@ -431,49 +409,6 @@ function writeVerbose(stream, msg, writeNewline = true) {
   }
 }
 
-const NAMESPACE_DIR_EXCEPTIONS = [
-  '.cli-metadata.json',
-  'node_modules',
-  'package.json',
-  '.eslintrc',
-];
-async function getLibDefNamespaces(defsDirPath: string) {
```

Comment:
Why where these methods removed?

Reply:
`getLocalLibDefs` is not used anywhere, which was only place using `getLibDefNamespaces `